src/main/java/com/zendesk/maxwell/schema/SchemaCapturer.java

```diff
@@ -48,111 +78,145 @@ public Schema capture() throws SQLException {
 		LOGGER.debug("Capturing schema");
 		ArrayList<Database> databases = new ArrayList<>();
 
-		ResultSet rs = connection.createStatement().executeQuery("SELECT * from INFORMATION_SCHEMA.SCHEMATA");
-
-		while ( rs.next() ) {
+		ResultSet rs = connection.createStatement().executeQuery(
+				"SELECT SCHEMA_NAME, DEFAULT_CHARACTER_SET_NAME FROM INFORMATION_SCHEMA.SCHEMATA"
+		);
+		while (rs.next()) {
 			String dbName = rs.getString("SCHEMA_NAME");
 			String charset = rs.getString("DEFAULT_CHARACTER_SET_NAME");
 
-			if ( includeDatabases.size() > 0 && !includeDatabases.contains(dbName))
+			if (includeDatabases.size() > 0 && !includeDatabases.contains(dbName))
 				continue;
 
-			if ( IGNORED_DATABASES.contains(dbName) )
+			if (IGNORED_DATABASES.contains(dbName))
 				continue;
 
-			databases.add(captureDatabase(dbName, charset));
+			Database db = new Database(dbName, charset);
+			databases.add(db);
 		}
+		rs.close();
+
+		int size = databases.size();
+		LOGGER.debug("Augmenting " + size + " databases with tables");
+		int counter = 1;
+		for (Database db : databases) {
+			LOGGER.debug(counter + "/" + size + " Augmenting " + db.getName());
+			long start = System.currentTimeMillis();
+			captureDatabase(db);
+			long end = System.currentTimeMillis();
+			LOGGER.debug(counter + "/" + size + " Augmented " + db.getName() + " after " + (end - start) + "ms");
+			counter++;
+		}
+		LOGGER.debug("Databases Augmented!");
```

Comment:
here too, ditch the timing information, no one will use it.  at some point I'll finish metrics and it can go there